pkg/metrics/metrics.go

```diff
@@ -455,6 +455,13 @@ var (
 		Name: "kvstore_operations_total",
 		Help: "Number of interactions with the Key-Value Store, labeled by subsystem, kind of action and action",
 	}, []string{LabelScope, LabelKind, LabelAction})
+
+	// FQDNGarbageCollectorCleanedTotal is the number of domains cleaned by the
+	// GC job.
+	FQDNGarbageCollectorCleanedTotal = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "fqdn_garbage_collector",
```

Comment:
I'm not sure that the name of this metric effectively communicates the meaning of the metric, one shouldn't need to read the help text to understand. There's conventions documented here which I think would be useful to follow: https://prometheus.io/docs/practices/naming/ . Ideally we'd have a bit of framework that adds to all the metrics and runs `promtool check metrics` against all of them, but for the moment I guess we run it locally.

Here's my naming suggestion, but I'm open to other suggestion too:

`fqdn_gc_deletions_total`

This communicates the area (fqdn), subarea (gc ie garbage collector, we've used this before for conntrack), type of operation (deletion) and unit (_total_ is the standard way to count integer units).